mlx_graphs/utils/validators.py

```diff
@@ -59,3 +60,24 @@ def wrapper(edge_index, edge_features=None, *args, **kwargs):
         return func(edge_index, edge_features, *args, **kwargs)
 
     return wrapper
+
+
+def validate_pandas_package(func):
```

Comment:
btw, such a decorator may be useful for your networkx imports @vinayhpandya, wdyt @francescofarina?

Reply:
Yes! perhaps you can make it parametric so that you can specify which package to validate?

Reply:
Yep! I just made it parametric.